addon/helpers/is-component-definition.js

```diff
@@ -0,0 +1,13 @@
+import { helper } from '@ember/component/helper';
+import { typeOf } from '@ember/utils';
+
+export function isComponentDefinition([ content ]) {
+  if (typeOf(content) !== 'object') {
+    return false;
+  }
+
+  let contentConstructorName = content.constructor.name || '';
+  return contentConstructorName.indexOf('ComponentDefinition') > -1;
```

Comment:
Should probably update `render-content` to use this function to keep things DRY 👍

Reply:
yeah, can do